tests/antithesis/test-template/robustness/traffic/main.go

```diff
@@ -159,10 +158,45 @@ func simulateTraffic(ctx context.Context, hosts []string, ids identity.Provider,
 		reports = append(reports, c.Report())
 		mux.Unlock()
 	}(compactClient)
+	for _, h := range hosts {
+		c := connect([]string{h}, ids, baseTime)
+		wg.Add(1)
+		go func(c *client.RecordingClient, lg *zap.Logger) {
+			defer wg.Done()
+			defer c.Close()
+			runDefragLoop(ctx, lg, c, traffic.DefaultCompactionPeriod, finish)
+			mux.Lock()
+			reports = append(reports, c.Report())
+			mux.Unlock()
+		}(c, lg)
+	}
 	wg.Wait()
 	return reports
 }
 
+func runDefragLoop(ctx context.Context, lg *zap.Logger, c *client.RecordingClient, period time.Duration, finish <-chan struct{}) {
+	for {
+		jittered := time.Duration(robustnessrand.RandRange(int64(period-period/2), int64(period+period/2)))
+		lg.Info("next defragment run", zap.String("host", c.Endpoints()[0]), zap.Time("at", time.Now().Add(jittered)))
```

Comment:
No need to log this, it will cause too much log volume, defrags should be visible in the report.